src/tree_data.h

```diff
@@ -477,13 +477,14 @@ struct lyd_meta {
  *
  * The union is used as a reference to the data's module and according to the format, it can be used as a key for
  * ::ly_ctx_get_module_implemented_ns() or ::ly_ctx_get_module_implemented(). While the module reference is always present,
- * the id member can be omitted in case it is not present in the source data as a reference to the default namespace.
+ * the id member can be omitted in case it is not present in the source data as a reference to the default module/namespace.
```

Comment:
s/id member/prefix member/ - but rather think about renaming the `name` member. In both cases, the structure is instantiated as `name` member. `name.name` seems weird. However, it is something I can live with.

Reply:
`id` should be fixed, but as for `name` member, I would say it is the best fitting name even though when accessing it it is repeated. Any ideas? Maybe the structure instances can be called something other than `name`, such as `oname` (opaque name) but I am not sure it would be an improvement.

Reply:
what about the `id` - imho it is the `identifier` from the YANG Grammar perspective

Reply:
I suppose, but these are data, so I am not sure it can really be called that way. In any case, this is a specific identifier `name` and I think it is always best to be as specific as possible. Also, the rest of the members are also identifiers.

Reply:
ok, as I said, I can live with it